data/categories.txt

```diff
@@ -1505,11 +1518,12 @@ el:Ηλεκτρονικά|κατάστημα
 sk:4Elektronika
 fa:فروشگاه تجهیزات الکترونیکی
 
-shop-hardware|shop-houseware|shop-doityourself
-en:4Hardware Store|DIY|do it yourself|U+1F3EC|U+1F3EA|U+1F50B|U+1F50C|U+1F4A1|U+1F526|U+1F529|U+1F528|U+2614
-ru:3Хозяйственный|Стройтовары|хозтовары|сделай сам
-bg:НСС|направи си сам|строителни материали
+shop-hardware|shop-houseware|shop-doityourself|@shop
+en:4Hardware|DIY|do it yourself|U+1F3EC|U+1F3EA|U+1F50B|U+1F50C|U+1F4A1|U+1F526|U+1F529|U+1F528|U+2614
+ru:3Хозяйственный|Строительный|Стройтовары|Хозтовары
 ar:متجر عدد وأدوات|أدوات|عدة|متجر
+be:Гаспадарчы|Будаўнічы
+bg:НСС|направи си сам|строителни материали
 cs:Železářství
 da:Isenkræmmer|Byggemarked
 nl:Ijzerhandel|Bouwmarkt
```

Comment:
nl:Ijzerhandel|Bouwmarkt|doe het zelf|ijzer

Reply:
Well, I removed "do it yourself" at all from this type. It may cause suspicious results.
So only "nl:Ijzerhandel|Bouwmarkt|ijzer"